test/distributed/_composable/fsdp/test_fully_shard_init.py

```diff
@@ -34,6 +38,7 @@ def world_size(self) -> int:
         return 1
 
     @unittest.skipIf(not TEST_CUDA, "no cuda")
+    @test_compiled_fsdp()
```

Comment:
I think if the unit test does not run forward, then there is no need to run with compile. What do you think?

Reply:
make sense. for full compile, we should also skip ?

Reply:
I think so. The only difference for full compile is to be careful of it changing the module structure (`_orig_mod`), which might affect state dict. Otherwise, the conditions for when to test compile should be the same.